llvm/lib/Transforms/Scalar/LoopIdiomRecognize.cpp

```diff
@@ -1064,53 +1073,95 @@ bool LoopIdiomRecognize::processLoopStridedStore(
     return Changed;
 
   // Okay, everything looks good, insert the memset.
+  // MemsetArg is the number of bytes for the memset libcall, and the number
+  // of pattern repetitions if the memset.pattern intrinsic is being used.
+  Value *MemsetArg;
+  std::optional<int64_t> BytesWritten = std::nullopt;
```

Comment:
Do we need `= std::nullopt`?

Reply:
No, I prefer being explicit but I guess this is more a personal preference and not consistently done within LLVM.